src/lib/util.ts

```diff
@@ -702,3 +703,6 @@ export function safeMoment(dateSpecifier: moment.Moment | Date | number | string
       throw new Error(`safeMoment: unknown type: ${typeof dateSpecifier}`);
   }
 }
+
+export const standardPolling = (fetchAllData = false) =>
```

Comment:
With this function you lose the ability to give different poll intervals for different polls, right?

Reply:
you don't lose it, you just have to write code to do it, which would be simple.  Currently that functionality is not required.